src/hooks/Recoil_Hooks.js

```diff
@@ -232,6 +233,11 @@ function useInterface(): RecoilInterface {
     function useRecoilState<T>(
       recoilState: RecoilState<T>,
     ): [T, SetterOrUpdater<T>] {
+      if (__DEV__) {
+        if (!isRecoilValue(recoilState))
```

Comment:
Looks like we don't have prettier running automatically on the GitHub side.  All if statements need blocks, single-statements aren't allowed.